packages/cmf/src/sagas/http.js

```diff
@@ -213,18 +235,68 @@ export function* httpGet(url, config, options) {
 	return yield* wrapFetch(url, config, HTTP_METHODS.GET, undefined, options);
 }
 
-export const handleDefaultConfiguration = curry((defaultConfig, config) =>
-	mergeCSRFToken(defaultConfig)(config),
+/**
+ * setDefaultHeader - define a default config to use with the saga http
+ * this default config is stored in this module for the whole application
+ *
+ * @param  {object} config key/value of header to apply
+ * @example
+ * import { setDefaultHeader } from '@talend/react-cmf/sagas/http';
+ * setDefaultHeader({headers: {
+ *  'Accept-Language': preferredLanguage,
+ * }});
+ */
+export function setDefaultConfig(config) {
+	let throwError = false;
+	if (defaultConfig) {
+		throwError = true;
+	}
+
+	defaultConfig = config;
+
+	if (throwError) {
+		throw new Error(
+			'ERROR: setDefaultConfig should not be called twice, if you wish to change the language use setDefaultLanguage api.',
+		);
+	}
```

Comment:
Please start with the if default config throw.
You can create an accessor to defaultConfig for testing purpose.